stdlib/influxdata/influxdb/alerts/alerts.flux

```diff
@@ -31,3 +32,29 @@ logs = (start, stop=now(), fn) =>
 deadman = (t, tables=<-) => tables
     |> max(column: "_time")
     |> map(fn: (r) => ( {r with dead: r._time < t} ))
+
+// levels describing the result of a check
+levelOK = 0
+levelInfo = 1
+levelWarn = 2
+levelCrit = 3
+levelUnknown = -1
```

Comment:
These need to be strings, it was decided we want users to be able to search for status using the string name of the level.